web-components/src/[sandbox]/examples/button-group.ts

```diff
@@ -0,0 +1,43 @@
+import "@/components/button-group/ButtonGroup";
+import "@/components/icon/Icon.ts";
+import { html } from "lit-element";
+
+export const buttonGroupTemplate = html`
+  <div class="column">
+    <h3>Icon only - 2 options</h3>
+    <md-button-group>
+      <button slot="button" type="button"><md-icon name="icon-text-table_16"></md-icon></button>
+      <button slot="button" type="button"><md-icon name="icon-analysis_16"></md-icon></button>
+    </md-button-group>
+  </div>
+  <div class="column">
+    <h3>Icon only - disabled</h3>
+    <md-button-group disabled>
+      <button slot="button" type="button"><md-icon name="icon-text-table_16"></md-icon></button>
```

Comment:
what will happen if i put test inside button content? how to make it screen-reader accessible? these recommendations need to be present here.

Reply:
Fixed